dogfood/data/images/jpg/main-img-preview.yaml

```diff
@@ -0,0 +1,3 @@
+file_name: images/jpg/main-img-preview.jpg
```

Comment:
Should this not be relative to the yaml?

Reply:
It's relative to /static so that there's no extra processing required to make an image tag.